client/components/Header/Header.js

```diff
@@ -117,7 +117,7 @@ class Header extends Component {
     return (
       <div className={styles.headerRoot}>
         <AppBar position="static" color="inherit" elevation={0}>
-          <Toolbar style={{ height: '128px' }}>
+          <Toolbar style={{height: '128px', 'zIndex': 1100}}>
```

Comment:
please, don't use zIndex to solve the problem you are having.
go to file Home.scss, and change this lines:
.homeContainer { top: 0  ... change to top:128px;
.homeGrid { margin-top: 128px ... remove margin-top prop. 
This will solve the problem without using zIndex :)